admin_ui/src/modules/meta.ts

```diff
@@ -1,23 +1,28 @@
-import axios from 'axios'
+import axios from "axios"
 
 export default {
     state: {
-        siteName: 'Piccolo Admin',
-        piccoloAdminVersion: 'Unknown'
+        siteName: "Piccolo Admin",
+        piccoloAdminVersion: "Unknown",
+        defaultLanguage: "english",
+        languages: undefined
```

Comment:
I think we can remove this as it is not being used.

Reply:
Good catch - will remove them!